src/root_console/components/AssignUsers.tsx

```diff
@@ -37,34 +42,34 @@ const AssignUsers = ({ users }) => {
     }
   }, []);
 
-  function handleProjectSelection(uuid) {
+  function handleProjectSelection(uuid: string) {
     // const selection = projects.find((project) => project.name === name);
     fetchUsersForProject(uuid);
   }
 
-  function handleUserSelection(uuid) {
-    setSelectedUser(uuid);
+  function handleUserSelection(uuid: string) {
+    setSelectedUserUuid(uuid);
     // const selection = users.find((user) => user.name )
   }
 
-  async function fetchUsersForProject(uuid) {
+  async function fetchUsersForProject(uuid: string) {
     try {
       console.log("project selection", uuid);
       const { data } = await getUsersForProject(uuid);
       console.log("users for project:", data);
       // const usersForProject = users.filter((user) => data.includes(user.id));
-      setCurrentUsers(data);
-      setSelectedProject(uuid);
+      setCurrentUsers(data.users);
```

Comment:
'data' is already fully deconstructed , so there’s no users property to reference on it. It will just hold the array of user objects.

Reply:
It's working on my end. This is the projectUsers type interface:
```typescript
export interface ProjectsUsersResponse {
  data: {
    users: {
      uuid: string;
      first_name: string;
      last_name: String;
    }
  };
  message: string;
}
```